native/packages/react-native-bpk-component-button/readme.md

```diff
@@ -69,3 +121,20 @@ export default class App extends Component {
 | large                 | bool                                                      | false    | false         |
 | selected              | bool                                                      | false    | false         |
 | type                  | oneOf('primary', 'featured', 'secondary', 'destructive')  | false    | null          |
```

Comment:
I wonder if we should still include the `theme` prop here but instead of listing a prop type, just link to the theme props heading i.e. `[See theme props below](#theme-props)`

Reply:
Yeah I was thinking that too - no harm in putting it in there.